.drone.yml

```
@@ -225,6 +226,86 @@ steps:
       kubernetes_token:
         from_secret: staging_kubernetes_token
 
+---
+depends_on: ["test-all"]
+kind: pipeline
+type: kubernetes
+name: dev-rel-build-chatbot-server-mongodb-public
+
+trigger:
+  event:
+    - tag
+  ref:
+    include:
+      - refs/tags/chatbot-server-mongodb-public-dev-rel-*
+
+steps:
+  # Builds and publishes Docker image for qa
+  - name: publish-dev-rel-chatbot-server-mongodb-public
+    image: plugins/kaniko-ecr
+    environment:
+      ENVIRONMENT: qa
```

```suggestion
      ENVIRONMENT: dev-rel
```

Do we need to update this?